Core/MIPS/x86/Jit.cpp

```diff
@@ -508,7 +455,7 @@ bool Jit::DescribeCodePtr(const u8 *ptr, std::string &name)
 			snprintf(temp, sizeof(temp), "%08x", jitAddr);
 		name = temp;
 	}
-	else if (asm_.IsInSpace(ptr))
+	else if (IsInSpace(ptr))
```

Comment:
Well, this is a duplicate to `name = "Unknown";` below.  Hopefully that never happens, but we only need one of the two.

-[Unknown]